src/utils/resources/udn/types.ts

```diff
@@ -5,8 +5,8 @@ export type UserDefinedNetworkAnnotations = {
 };
 
 export enum UserDefinedNetworkRole {
-  Primary = 'Primary',
-  Secondary = 'Secondary',
+  primary = 'primary',
+  secondary = 'secondary',
 }
```

Comment:
in `UserDefinedNetworkLayer2` the `role` have the first letter uppercase like `Primary` and not `primary` . We can create another enum or use the `.toLowerCase`